Source/DafnyCore/Rewriters/ExpectContracts.cs

```diff
@@ -222,34 +221,42 @@ public class ExpectContracts : IRewriter {
   }
 
   /// <summary>
-  /// Add wrappers for certain top-level declarations in the given module.
-  /// This runs after the first pass of resolution so that it has access to
-  /// ghostness information, attributes and call targets.
+  /// Adds wrappers for certain top-level declarations in the given
+  /// program and redirects callers to call those wrappers instead of
+  /// the original members.
+  ///
+  /// This runs after resolution so that it has access to ghostness
+  /// information, attributes and call targets. It runs on the entire
```

Comment:
The sentence 'it runs on the entire program' doesn't make sense to me.

I think this is a transformation where modules are only affected by modules they import, but not the other way around, so you can do it module by module, which the existing loops indicate. I think you can cache the results of processing a module, even though we don't need to do that now.

Reply:
I guess that comment isn't as relevant as it was before. When running after resolution, there are two possible methods to override, and this was an explanation for why the code overrides the `Program` version instead of the `Module` version. After verification, there's only one option, though.